### DISCUSSION TITLE
Browser Mod pop-up window

### DISCUSSION THREAD

**Discussion by marsal-75** [kalkih/mini-media-player#837] [Q&A]
I'd like to show a card containing the TV remote control in a pop-up by clicking on a shortcut using fire-dom-event. I've read that it should be possible but I haven't been able to do it in any way.
This is my attempt, where am I wrong?
```
entity: media_player.tv_salone
type: custom:mini-media-player
icon: mdi:television
info: scroll
artwork: material
toggle_power: false
hide:
  volume: true
  source: true
shortcuts:
  columns: 1
  buttons:
    - icon: mdi:remote
      type: action
      id: tap_action
      action: fire-dom-event
      browser_mod:
        service: browser_mod.popup
        data:
          content:
            type: entity
            entity: media_player.tv_salone
```
PS:
Don't pay attention to the entity contained in the pop-up: it's just to shorten the code to make attempts without having to modify the 
indentation every time.
Thank you in advance.

**Comment by madjetey** [comment]
Did you ever make any progress with this?  Stuck trying to pull the same thing off

**Reply by marsal-75** [reply]
Yes, I solved it by setting a tap action on the card icon itself since it didn't work from the shortcuts. I did it in this way:
```
entity: media_player.tv_salone
type: custom:mini-media-player
icon: mdi:remote-tv
info: scroll
artwork: material
toggle_power: true
hide:
  volume: true
  source: true
tap_action:
  action: fire-dom-event
  browser_mod:
    service: browser_mod.popup
    data:
      content:
        ----
        ----
```
